google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml

```diff
@@ -0,0 +1,162 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.devtools.clouddebugger.spi.v2
+  python:
+    package_name: google.cloud.gapic.devtools.clouddebugger.v2
```

Comment:
I had originally hoped that this could be derived from the [`title` attribute](https://github.com/googleapis/googleapis/pull/145/files/0415ff5d6d0d2ede2dfa54b117fc6341de4c7939#diff-33c86e384db847920bc6861fd089906eR6) of the service config via googleapis/toolkit#270. @garrettjonesgoogle also has a proposed introducing a "short name" attribute into configuration for deriving the output directories, since it may not be possible to automatically derive this from service config -- if that happens, I'd assume it would coincide with `{api}` here as well?